modules/platforms/dotnet/Apache.Ignite.Core.Tests/Cache/Query/Linq/CacheLinqTest.Misc.cs

```diff
@@ -127,6 +127,9 @@ public void TestScalarQuery()
 
             Assert.AreEqual(PersonCount, cache.Count());
             Assert.AreEqual(PersonCount, cache.Count(x => x.Key < PersonCount));
+            
+            Assert.That(cache.LongCount(), Is.EqualTo(PersonCount));
```

Comment:
Please use `Assert.AreEqual` to be consistent with the rest of the tests.

Reply:
Fixed. Thanks.